ros2_control/package.xml

```diff
@@ -16,6 +16,7 @@
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>controller_parameter_server</exec_depend>
   <exec_depend>hardware_interface</exec_depend>
+  <exec_depend>test_robot_hardware</exec_depend>
```

Comment:
I'd opt for not adding testing packages to the metapackage as it doesn't form the main part of what we'd like someone to install when they install `ros2_control`. It's good to keep the installed bundle at a reasonable size.

Reply:
we can add them as a test dependency, but I generally don't agree with not listing them. If we build up to `ros2_control` all test packages should be compiled as well.

Reply:
Thanks